core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -120,13 +120,13 @@ abstract contract FeePayer is Testable, Lockable {
         // If the effective fees paid as a % of the pfc is > 100%, then we need to reduce it and make the contract pay as much of the fee
         // that it can (up to 100% of its pfc). We'll reduce the late penalty first and then the regular fee, which has the effect of paying
         // the store first, followed by the caller if there is any fee remaining.
-        if (totalPaid.isGreaterThan(_pfc)) {
```

Comment:
How did this pass ci?

Reply:
I have no idea? it failed immediately post-merge?

Reply:
merge conflicts somehow i'm guessing

Reply:
how are we having things pass CI that we think should fail? it's a bit worrying.

Reply:
Its happening when PR 1 is approved, and then another PR 2 is merged, and then the PR 1 is merged into master, which github auto-resolves any merge conflicts between PR 1 and 2.

Perhaps we should force manual merges with master?